src/ffi/dial_ffi.rs

```diff
@@ -168,9 +168,12 @@ pub unsafe extern "C" fn dial(
     // if the uri is local then we can connect directly.
     let disable_webrtc;
     if let Some(host) = uri.host() {
-        disable_webrtc = host.contains(".local") || host.contains("localhost");
+        disable_webrtc =
+            host.contains(".local") || host.contains("localhost") || host.contains("0.0.0.0");
```

Comment:
do we also want to catch 127.0.0.1? in that case we main need to use a Regex

Reply:
I think we could probably write a function with all the matching text in a single `HashSet` and then iter over it to find matches. Avoids excessive duplication and also a regex, which I'm not sure is necessary at this point.

Reply:
that works for me, me need to make sure to exclude address containing localhost (if someone names his robot localhost :P)

Reply:
oh that would be sad :( we could match on `localhost:` instead?